suite/tests/CMakeLists.txt

```diff
@@ -1459,6 +1459,18 @@ function(torun test key source native standalone_dr dr_ops exe_ops added_out)
     # global reg keys and can thus run multiple instances of the same
     # app name simultaneously so we do not need to mark RUN_SERIAL
   endif (WIN32)
+
+  # A subset of release tests has been added and labeled with RUN_IN_RELEASE. It focuses
+  # on Google's use of the drcachesim client. We do not want to extend the Travis
+  # regression time by too much and keep the following list concise.
```

Comment:
What is the time increase with this new list?

Reply:
(Not asking for code changes -- wish github had a checkbox for that.)

Reply:
The newly added tests run about 2s total.